Makefile

```diff
@@ -21,16 +21,13 @@ run-flask:
 test:
 	flake8 .
 	isort --check-only ./app ./tests
-	npm test
+	source $(HOME)/.nvm/nvm.sh && npm test
```

Comment:
Would it make more sense to put this in the CI pipeline before we run `make`? It's not necessary when running it locally so would be easy to make this same mistake again.

Reply:
Tried it. It's not available to Make in a non-interactive shell so has to be "re-sourced". What you suggest works fine locally (since it's interactive).

Reply:
Oh that's a shame - maybe we should have stuck with `n` 😆 .

Reply:
Yes I think n just works using go-ol' symlinks